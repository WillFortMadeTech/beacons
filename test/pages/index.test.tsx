import React from "react";
import { render, screen } from "@testing-library/react";

import ServiceStartPage from "../../src/pages";

describe("ServiceStartPage", () => {
  it("directs the user to the check your beaecon details page", () => {
    render(<ServiceStartPage />);

    expect(screen.getByRole("button")).toHaveAttribute(
      "href",
      "/register-a-beacon/check-beacon-details"
    );
  });
});
