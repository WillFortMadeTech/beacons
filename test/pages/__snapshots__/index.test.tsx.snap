// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page renders correctly 1`] = `
<DocumentFragment>
  <div
    class="govuk-width-container"
  >
    <main
      class="govuk-main-wrapper"
    >
      <h1
        class="govuk-heading-l"
      >
        Beacon registration service
      </h1>
      <p
        class="govuk-body"
      >
        Use this service to:
      </p>
      <ul
        class="govuk-list govuk-list--bullet"
      >
        <li>
          <a
            href="https://github.com/madetech/mca-beacons-webapp/actions"
            rel="noreferrer"
            target="_blank"
          >
            showcase
          </a>
           your 
          <a
            href="https://github.com/madetech/mca-beacons-integration/actions"
            rel="noreferrer"
            target="_blank"
          >
            continuous integration and deployment pipeline
          </a>
        </li>
        <li>
          talk about reusable 
          <a
            href="https://design-system.service.gov.uk/"
            rel="noreferrer"
            target="_blank"
          >
            GOV.UK design system
          </a>
           components that will speed up future sprints
        </li>
        <li>
          outline the 
          <a
            href="https://miro.com/app/board/o9J_lZRBWY4=/"
            rel="noreferrer"
            target="_blank"
          >
            data model
          </a>
           as we currently understand it
        </li>
      </ul>
    </main>
  </div>
</DocumentFragment>
`;
